modules/ROOT/pages/security/ssl-fips-compatibility.adoc

```diff
@@ -0,0 +1,130 @@
+= Configure SSL for FIPS 140-2 compatibility
+
+The National Institute of Standards and Technology (NIST) developed the Federal Information Processing Standard (FIPS) Publication 140-2 as a security standard that sets forth requirements for cryptographic modules, including hardware, software, and/or firmware, for U.S. federal agencies.
+While FIPS 140 compliance is primarily required for federal agencies and their contractors, it also is used in the healthcare sector under regulations like the Health Insurance Portability and Accountability Act (HIPAA) to protect patient data.
+
+Neo4j 4.4 officially supports FIPS 140-2 compliance.
+
+This guide helps configure Neo4j to use TLS/SSL encryption in a FIPS-compliant way.
+It is supplementary to the xref:security/ssl-framework.adoc[SSL framework documentation], as many of the configuration processes and requirements are the same.
+
+
+[[fips-prerequisites]]
+== Prerequisites
+
+* Verify that the machine running Neo4j has FIPS-compatible hardware and operating system.
+Only xref:installation/requirements.adoc#deployment-requirements-software[Linux operating systems] are supported for Neo4j FIPS compatibility.
+* Deploy OpenSSL 3.0.8/3.0.9 in the FIPS mode.
+* Install and configure a non-native authentication provider, for example LDAP or SSO.
+See xref:authentication-authorization/index.adoc[].
+
+
+[[fips-ssl-providers]]
+== Enable FIPS SSL provider
+
+Set SSL policies on Bolt and HTTPS in the _neo4j.conf_ file.
+See xref:security/ssl-framework.adoc#ssl-configuration[SSL framework -> Configuration] for detailed instructions.
+
+=== Set SSL provider to OpenSSL
+
+Instructions on how to build and install a FIPS-compatible OpenSSL are out of scope for this document.
+Installation steps can differ depending on operating system, and other security requirements you might have for OpenSSL.
+
+* For a list of FIPS certified OpenSSL versions, see link:https://openssl-library.org/source/[].
+* A FIPS provider must be installed into OpenSSL.
+* OpenSSL must be configured to use the FIPS provider by default.
+
+=== Use the correct version of the `netty-tcnative` library
+
+To achieve FIPS compliance, you must use the dynamically linked version of `netty-tcnative` alongside a FIPS-compatible installation of OpenSSL.
+
+. The dynamically linked library requires the Apache Portable Runtime Library also to be installed.
+To install it, use the operating system’s package manager.
++
+In Debian/Ubuntu this package is usually called `libapr1`.
++
+[source, console, subs="attributes"]
+.Install Apache Portable Runtime Library in Debian or Ubuntu
+----
+apt install -y libapr1
+----
++
+In RedHat Enterprise Linux, the package is usually called `apr`:
++
+[source, console, subs="attributes"]
+.Install Apache Portable Runtime Library in RedHat
+----
+dnf install -y apr
+----
++
+. Neo4j compatible JAR files can be found at:
+https://assets.neo4j.com/netty-tcnative/2.0.65.Final/simplified/netty-tcnative-2.0.65.Final-all.tar
```

Comment:
Should we do the same for the 5.x docs?

Reply:
No I don't think so. The jars are already included in 5x neo4j so there should be no need to download them separately.